helpdesk/__manifest__.py

```diff
@@ -0,0 +1,41 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Helpdesk',
+    'summary': """
+        Helpdesk""",
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'After-Sales',
+    'author':   'AdaptiveCity, '
```

Comment:
@emagdalenaC2i this alignment is not suitable in Python. Take it into account for next time. I'm correcting it on my rebase.

Reply:
Ok, noted